js/src/components/CollapsibleCornerstone.js

```diff
@@ -14,16 +18,14 @@ import CornerstoneToggle from "yoast-components/composites/Plugin/CornerstoneCon
  * @constructor
  */
 export default function CollapsibleCornerstone( { isCornerstone, onChange } ) {
-	const { makeOutboundLink } = utils;
-	const LearnMoreLink = makeOutboundLink();
-
 	return (
 		<Collapsible title={ __( "Cornerstone content", "wordpress-seo" ) }>
-			<p>{ __( "Cornerstone content should be the most important and extensive articles on your site. ", "wordpress-seo" ) }
+			<HelpText>
+				{ __( "Cornerstone content should be the most important and extensive articles on your site. ", "wordpress-seo" ) }
```

Comment:
We shouldn't use trailing spaces in the translatable strings, as in 99% of the cases they will be lost during translations. We can't rely on translators to take care of trailing spaces.